src/interface/drawstatus.cpp

```diff
@@ -430,35 +430,52 @@ void drawStatus()
 		//drawImage(textdown_bmp, NULL, &pos);
 	}*/
 
-	// health
+	// PLAYER HEALTH BAR
+	// Display Health bar border
 	pos.x = 76;
 	pos.w = 38;
 	pos.h = 156;
 	pos.y = yres - 168;
 	drawTooltip(&pos);
+
+	// Display "HP" at top of Health bar
 	ttfPrintText(ttf12, pos.x + 8, pos.y + 6, language[306]);
 
+	// Display border between actual Health bar and "HP"
+	pos.x = 76;
+	pos.w = 38;
+	pos.h = 1;
```

Comment:
Change height from 1 to 0 - Makes the border 2px high instead of 3px. Matches the 2px border for the rest of the bar.